typings/index.d.ts

```diff
@@ -1458,6 +1458,7 @@ declare module 'klasa' {
 
 		export interface DMChannel extends SendAliases, ChannelExtendables {}
 
+		export interface NewsChannel extends SendAliases, ChannelExtendables {}
```

Comment:
Please add some spacing:
```suggestion
		export interface NewsChannel extends SendAliases, ChannelExtendables {}

```